SU-SWS/stanford_components
# EARTH-401 added image captions to simple block & slat

# READY FOR REVIEW

# Summary
- Refactored simple slat, simple slats, and simple block.
- Added caption field for the image
- wrapped the image and caption in `<figure>` tag
- added variants for simple slat.

# Needed By (Date)
- End of sprint

# Urgency
- Medium

# Steps to Test

1. Checkout branch and same branch in [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/80)
2. clear cache. enable stanford_paragraph_slat module
3. review patterns/simple_slat, patterns/simple_slats, & patterns/simple_block pages to verify they still look the same
4. create complex page with "Editorial Sidebar" component
5. verify it looks good enough for now.

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_paragraph_types/pull/80

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
